<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_qr_payment</class>
 <widget class="QWidget" name="page_qr_payment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_previous_page">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_price">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>1470</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
font-style: light;
font-size: 24px;
color: #58595B;</string>
   </property>
   <property name="text">
    <string>label_product_price</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_product_information">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>1470</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Montserrat;
font-style: light;
font-size: 24px;
color: #58595B;</string>
   </property>
   <property name="text">
    <string>order summary</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_qrCode">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>360</width>
     <height>360</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_refresh_page">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>1340</y>
     <width>861</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FF0000;
font-family: Montserrat;
font-style: light;
font-weight: bold;
font-size: 40px;
line-height: 29px;</string>
   </property>
   <property name="text">
    <string>timeouttext</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>1931</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_processing">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>1080</width>
     <height>251</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>processing label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_steps">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>940</y>
     <width>980</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string>steps</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>1080</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_scan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>1080</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>scan label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_gif">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>960</y>
     <width>350</width>
     <height>350</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_qr_background">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>349</y>
     <width>600</width>
     <height>706</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_qr_background</zorder>
  <zorder>pushButton_refresh</zorder>
  <zorder>pushButton_previous_page</zorder>
  <zorder>label_product_price</zorder>
  <zorder>label_product_information</zorder>
  <zorder>label_qrCode</zorder>
  <zorder>label_refresh_page</zorder>
  <zorder>label_steps</zorder>
  <zorder>label_title</zorder>
  <zorder>label_scan</zorder>
  <zorder>label_processing</zorder>
  <zorder>label_gif</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
